html/pfappserver/root/src/views/Configuration/pki/profiles/_components/TheForm.vue

```diff
@@ -119,6 +119,38 @@
           :text="$i18n.t('Cloud Service to integrate.')"
         />
       </base-form-tab>
+      <base-form-tab :title="$i18n.t('Renewal Configuration')">
+        <form-group-days-before-renewal namespace="days_before_renewal"
+          :column-label="$i18n.t('Days before renewal')"
+          :text="$i18n.t('Number of days before the pki authorize renewal. Set to 0 mean renewal is always allowed')"
```

Comment:
Number of days before the pki authorize renewal. Set to 0 mean renewal is always allowed => Number of days before the PKI authorizes renewal. Setting it to 0 means renewal is always allowed.